tools/installers/install_warp-rnnt.sh

```diff
@@ -10,22 +10,9 @@ fi
 
 cuda_ver=$1
 
-if [ $cuda_ver != "10.0" ]; then
-    echo "warp-rnnt was only tested with CUDA_VERSION=10.0. Skipping install."
+if [ $cuda_ver != "10.0" ] || [ $cuda_ver != "10.1" ] || [ $cuda_ver != "10.2" ]; then
```

Comment:
```suggestion
if ! [[ "$cuda_ver" =~ ^(10.0|10.1|10.2)$ ]]; then
```

I tested all cuda versions, I'm surprised you could install repo with `CUDA_VERSION=10.1` or `10.2`?

Reply:
I can't say I thoroughly tested it, but the warp-rnnt unit test worked. Though I installed warp-rnnt with pip, if it matters.

Reply:
No, it is `python -m warp_rnnt.test`

Reply:
Sorry I read too quickly!
Hum, every install (from repo or pypi) with a cuda version != 10.0 ended-up with an ImportError when I tested. Could you verify? If not, I'll do it tomorrow

Reply:
Welp, I have no ImportError in cuda 10.1 environment. I'm going to try this also with a clean miniconda environment to verify though.

Reply:
Sorry, I edited my message `ImportError` -> `error`! The error I encountered was:

```
  File "/home/fboye/espnet/tools/venv/lib/python3.7/site-packages/warp_rnnt-0.4.0-py3.7-linux-x86_64.egg/warp_rnnt/__init__.py", line 2, in <module>
    import warp_rnnt._C as core
ImportError: /home/fboye/espnet/tools/venv/lib/python3.7/site-packages/warp_rnnt-0.4.0-py3.7-linux-x86_64.egg/warp_rnnt/_C.cpython-37m-x86_64-linux-gnu.so: undefined symbol: cudaSetupArgument
```

Reply:
Just checked with clean cuda 10.2 torch 1.6 env. I am using Ubuntu 18.04.
```
import warp_rnnt._C as core
...
costs, grads = core.rnnt_loss(
...             xs.cuda(), ys.cuda(),
...             xn.cuda(), yn.cuda())

>>> costs
tensor([4.2742], device='cuda:0')
>>> grads
tensor([[[[ 0., -1.,  0.,  0.,  0.],
          [ 0.,  0., -1.,  0.,  0.],
          [-1.,  0.,  0.,  0.,  0.]]]], device='cuda:0')```

Reply:
Could you also try with torch==1.4 please? If it works, let's keep that condition (with the fix I proposed or change `||` to `&&`)!
I may have made a mistake when testing, good to know if it also works for other cases!

Thanks!

Reply:
Sorry, I pushed the fix just before reading your last comment.
torch 1.4 was my initial setup, where everything worked. I just used 1.6 for a wider check.

Reply:
OK, thank you!